scripts/globals/mobskills/impulse_drive.lua

```diff
@@ -30,14 +30,6 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
             target:addStatusEffect(xi.effect.BIND, 1, 0, duration)
         end
     end
-
-    if
-        dmg > 0 and
-        skill:getMsg() ~= 31
-    then
-        target:tryInterruptSpell(mob, info.hitslanded)
-    end
-    return dmg
```

Comment:
Did you intend to remove the return as well?

Reply:
Whoops 👁️ 👁️